AppBidding-MoPub/AppBidding-MoPub/HomeViewController.h

```diff
@@ -0,0 +1,17 @@
+//
+//  HomeViewController.h
+//  AppBidding-MoPub
+//
+//  Copyright © 2020 Criteo. All rights reserved.
+//
+
+
+#ifndef HomeViewController_h
```

Comment:
Same comment than in BannerViewController.h: by using the #import directive, you do not need such protection.